Makefile

```diff
@@ -22,6 +22,7 @@ export LC_ALL
 
 include build/makelib/common.mk
 include build/makelib/helm.mk
+include build/makelib/linters.mk
```

Comment:
I would prefer not to have more of a proliferation of makefiles than we already have. Since the added targets are short, maybe we can reuse an existing file?

Reply:
I would add it to the main Makefile.

Reply:
done